quickstarts/kit/custom-istio-enricher/app/pom.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2019 Red Hat, Inc.
+    This program and the accompanying materials are made
+    available under the terms of the Eclipse Public License 2.0
+    which is available at:
+
+        https://www.eclipse.org/legal/epl-2.0/
+
+    SPDX-License-Identifier: EPL-2.0
+
+    Contributors:
+      Red Hat, Inc. - initial API and implementation
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>eclipse-jkube-sample-custom-enricher-app</artifactId>
+  <packaging>jar</packaging>
+
+  <parent>
+    <groupId>org.eclipse.jkube.quickstarts.kit</groupId>
+    <artifactId>eclipse-jkube-sample-custom-enricher-parent</artifactId>
+    <version>1.1.0-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+
+  <name>Eclipse JKube :: Quickstarts :: Kit :: Custom Enricher App</name>
+  <description>Spring Boot example with a custom enricher</description>
```

Comment:
This description goes into the website, try to make it a little bit more thorough.
Maybe bring here the first two paragraphs of the README.
Take into account that (for now), line breaks are propagated to the website. Please keep each paragraph in a single line.

Reply:
I see this is multi-module. Add the valid description to the parent module (in case it's not there).